packages/application-components/src/components/internals/page.styles.ts

```diff
@@ -3,6 +3,7 @@ import { customProperties } from '@commercetools-uikit/design-system';
 
 export const ContentWrapper = styled.div`
   flex: 1;
+  flex-basis: 0;
```

Comment:
Flex-basis overrules any specified CSS width value, so if you set flex-basis to 0, it doesn't fall back to the CSS "width" value.